setup.py

```diff
@@ -18,14 +51,7 @@
     # We are including the tests because other libraries do use mixins from them.
     packages=find_packages(),
     license='AGPL-3.0',
-    install_requires=[
-        'six>=1.10.0,<2.0.0',
-        'stevedore>=0.14.1,<2.0.0',
-        'pymongo>=2.7.2,<4.0.0'
-    ],
-    extras_require={
-        'django': ['Django>=1.11,<2.0;python_version<"3"', 'Django>=1.11;python_version>"3"']
-    },
```

Comment:
@timmc-edx: Did you ever find out why this was pulled out as `extra_requires`, and whether or not that matters?

Reply:
No, I can't figure out why it was like that, although I'm still really hazy on how extras work in Python dep management. And it looks like Django never made it into the requirements files, so maybe this needs another (small) PR.

It looks like maybe this is what we want: https://github.com/edx/mockprock/blob/181d2c962006ffcecfda009ee9b6f417982c48c6/setup.py#L61

Reply:
What you found looks good if you want to keep the same.  The commit that added it is https://github.com/edx/opaque-keys/commit/c056e200fd8c21819a31d1f6025290521073b2da, and Alex still works here.  So, you could either ask Alex or members of BOM that know more. :)